spec/src/main/asciidoc/chapters/introduction/entities.asciidoc

```diff
@@ -0,0 +1,191 @@
+== Type system
+
+This specification assumes that data is represented in a structured form, that is, that it is expressed in terms of:
+
+1. **atomic values**,
```

Comment:
In Jakarta Data, I remember that we used `basic type`
Does it make sense to put the same name here?

Reply:
Well, that's what @njr-11 suggested too. And it's the language that's been traditionally used in the JPA spec.

But I just think it's not a great term, and I think we should consider changing it. "Atomic" is much more consistent with the terminology generally used in the PL literature.

Reply:
If we rename both, I am fine.

My concern is more about having the same terminology for consistency.